R/addins.R

```diff
@@ -3,7 +3,15 @@
 #' Helper function for RStudio Addin.
 style_active_file <- function() {
   context <- get_rstudio_context()
-  style_file(context$path, style = tidyverse_style)
+  if (context$path != "") {
```

Comment:
What's the advantage of case distinction here? Can't we just use `rstudioapi` to handle all cases?

Reply:
I just assumed I should avoid changing the current behaviour for the case when the active file has been saved (i.e that it is styled and saved).

Was your intention with #235 to just modify (without saving) in all cases?

Reply:
For consistency, I suggest to stick with `rstudioapi` throughout. See https://github.com/krlmlr/styler/issues/235#issuecomment-336376681.